### PR TITLE
[MMCA-4257] Update version & configuration for CDS Financials service

### PR DESCRIPTION
**Pull request by wg-hmrc**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,4 +19,6 @@ dist
 node_modules/
 npm-debug.log
 yarn-debug.log
-yarn-error.log
\ No newline at end of file
+yarn-error.log
```
Comment (AdamCoops) [inline .gitignore:22]: Is this change intentional?

Comment (wg-hmrc) [reply .gitignore:22]: Hey @AdamCoops  yes its intentional, those files were coming from my IDE

**Review by tyagikumarvineet — APPROVED** [review]

**Review by AdamCoops — APPROVED** [review]
